python/tank/bootstrap/resolver.py

```diff
@@ -194,9 +204,75 @@ def resolve_configuration(self, config_descriptor, sg_connection):
                 self._bundle_cache_fallback_paths
             )
 
+    def find_matching_pipelines(self, pipeline_config_name, current_login, sg_connection):
+        """
+        Finds all the pipelines matching the query parameters.
```

Comment:
At line 102, you have (from earlier)

```
raise TankBootstrapError("Cannot locate legacy pipeline configuration at '%s'!" % (config_path,))
```

Sorry must have missed this in previous CR. Could it just say something like

```
raise TankBootstrapError("Installed pipeline configuration '%s' does not exist on disk!" % (config_path,))
```